[apr] Build error /vcpkg/packages/apr_x64-linux/debug/lib/pkgconfig/apr-1.pc   file failed to open for reading (No such file or directory):

**Host Environment**
- OS: [e.g. Windows Server 2019/ WSL Linux 20.04 LTS]
- Compiler: gcc

**To Reproduce**
Steps to reproduce the behavior:
```
/usr/bin/cmake -G "Ninja"   -DCMAKE_BUILD_TYPE:STRING="Debug" -DCMAKE_INSTALL_PREFIX:PATH="../cmake-install/WSL-GCC-Debug" -DCMAKE_TOOLCHAIN_FILE:FILEPATH="/home/user/vcpkg/scripts/buildsystems/vcpkg.cmake" -  CMakeLists.txt;
```
```
1> [CMake] *** portable models build ***
1> [CMake] -- Running vcpkg install
1> [CMake] Detecting compiler hash for triplet x64-linux...
1> [CMake] The following packages will be built and installed:
1> [CMake]   * apr[core]:x64-linux -> 1.7.0#4 -- /home/ec2-user/vcpkg/buildtrees/versioning_/versions/apr/b9e93acdaa680398eaed361f1df530096ded84ff
1> [CMake]   * apr-util[core]:x64-linux -> 1.6.1#4 -- /home/ec2-user/vcpkg/buildtrees/versioning_/versions/apr-util/cf1a0e97d00a5748dc58db7d0a5da0fd7d5efb66
1> [CMake]     log4cxx[core]:x64-linux -> 0.11.0#3 -- /home/ec2-user/vcpkg/buildtrees/versioning_/versions/log4cxx/406017355e07d94a41999c3db5e015be8bf0fbbd
1> [CMake]     tbb[core]:x64-linux -> 2020_U3#6 -- /home/ec2-user/vcpkg/buildtrees/versioning_/versions/tbb/e168e03c2e5776bdee59accf5f643ae26b0ddd84
1> [CMake]     tinyxml2[core]:x64-linux -> 9.0.0 -- /home/ec2-user/vcpkg/buildtrees/versioning_/versions/tinyxml2/a8b086bdb3eaa94fea466bf8565d7c98b00c9f94
1> [CMake] Additional packages (*) will be modified to complete this operation.
1> [CMake] Restored 0 packages from /home/ec2-user/.cache/vcpkg/archives in 1.139 ms. Use --debug to see more details.
1> [CMake] Starting package 1/5: apr:x64-linux
1> [CMake] Building package apr[core]:x64-linux...
1> [CMake] -- Installing port from location: /home/ec2-user/vcpkg/buildtrees/versioning_/versions/apr/b9e93acdaa680398eaed361f1df530096ded84ff
1> [CMake] -- Using cached apr-1.7.0.tar.bz2.
1> [CMake] -- Cleaning sources at /home/ec2-user/vcpkg/buildtrees/apr/src/apr-1-3a6fcafcf9.clean. Use --editable to skip cleaning for the packages you specify.
1> [CMake] -- Extracting source /home/ec2-user/vcpkg/downloads/apr-1.7.0.tar.bz2
1> [CMake] -- Using source at /home/ec2-user/vcpkg/buildtrees/apr/src/apr-1-3a6fcafcf9.clean
1> [CMake] -- Configuring apr
1> [CMake] -- Getting CMake variables for x64-linux-dbg
1> [CMake] -- Getting CMake variables for x64-linux-rel
1> [CMake] -- Configuring x64-linux-rel
1> [CMake] -- Building x64-linux-rel
1> [CMake] -- Installing x64-linux-rel
**1> [CMake] CMake Error at scripts/cmake/vcpkg_replace_string.cmake:12 (file):
1> [CMake]   file failed to open for reading (No such file or directory):
1> [CMake] 
1> [CMake]     /home/ec2-user/vcpkg/packages/apr_x64-linux/debug/lib/pkgconfig/apr-1.pc
1> [CMake] Call Stack (most recent call first):**
1> [CMake]   buildtrees/versioning_/versions/apr/b9e93acdaa680398eaed361f1df530096ded84ff/portfile.cmake:81 (vcpkg_replace_string)
1> [CMake]   scripts/ports.cmake:145 (include)
1> [CMake] Error: Building package apr:x64-linux failed with: BUILD_FAILED
1> [CMake] Please ensure you're using the latest portfiles with `git pull` and `./vcpkg update`.
1> [CMake] Then check for known issues at:
1> [CMake]   https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+apr
1> [CMake] You can submit a new issue at:
1> [CMake]   https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[apr]+Build+error
1> [CMake] including:
1> [CMake]   package: apr[core]:x64-linux -> 1.7.0#4
1> [CMake]     vcpkg-tool version: 2022-03-09-1affd32f93b299d5a907816c328ca3ededb73a7e
1> [CMake]     vcpkg-scripts version: 6308196b2 2022-03-09 (17 hours ago)
1> [CMake] 
1> [CMake] Additionally, attach any relevant sections from the log files above.
1> [CMake] -- Running vcpkg install - failed
1> [CMake] CMake Error at /home/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake:834 (message):
1> [CMake]   vcpkg install failed.  See logs for more information:
1> [CMake]  /cmake-build/WSL-GCC-Debug/vcpkg-manifest-install.log
1> [CMake] Call Stack (most recent call first):
1> [CMake]   /usr/share/cmake-3.19/Modules/CMakeDetermineSystem.cmake:123 (include)
1> [CMake]   CMakeLists.txt:31 (project)
1> [CMake] CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
1> [CMake] CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
1> [CMake] CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
1> [CMake] -- Configuring incomplete, errors occurred!
```
**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

--- a/ports/apr/portfile.cmake
+++ b/ports/apr/portfile.cmake
@@ -73,9 +73,12 @@ else()
 
     vcpkg_install_make()
 
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/apr-1.pc"
-        "-lapr-\${APR_MAJOR_VERSION}" "-lapr-1"
-    )
+    if(NOT VCPKG_BUILD_TYPE)
+        vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/apr-1.pc"
+            "-lapr-\${APR_MAJOR_VERSION}" "-lapr-1"
+        )
+    endif()
+
     vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/lib/pkgconfig/apr-1.pc"
         "-lapr-\${APR_MAJOR_VERSION}" "-lapr-1"
     )
@@ -84,11 +87,13 @@ else()
     vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin/apr-1-config" "\"${CURRENT_INSTALLED_DIR}\"" "`dirname $0`/../../..")
     vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin/apr-1-config" "APR_SOURCE_DIR=\"${SOURCE_PATH}\"" "")
     vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin/apr-1-config" "APR_BUILD_DIR=\"${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel\"" "")
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin/apr-1-config" "\"${CURRENT_INSTALLED_DIR}/debug\"" "`dirname $0`/../../../..")
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin/apr-1-config" "APR_SOURCE_DIR=\"${SOURCE_PATH}\"" "")
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin/apr-1-config" "APR_BUILD_DIR=\"${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg\"" "")
+    
+    if(NOT VCPKG_BUILD_TYPE)
+        vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin/apr-1-config" "\"${CURRENT_INSTALLED_DIR}/debug\"" "`dirname $0`/../../../..")
+        vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin/apr-1-config" "APR_SOURCE_DIR=\"${SOURCE_PATH}\"" "")
+        vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/${PORT}/debug/bin/apr-1-config" "APR_BUILD_DIR=\"${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg\"" "")
+    endif()
 endif()
 
 # Handle copyright
 file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
-

--- a/ports/apr/vcpkg.json
+++ b/ports/apr/vcpkg.json
@@ -1,9 +1,10 @@
 {
   "name": "apr",
   "version": "1.7.0",
-  "port-version": 7,
+  "port-version": 8,
   "description": "The Apache Portable Runtime (APR) is a C library that forms a system portability layer that covers many operating systems.",
   "homepage": "https://apr.apache.org/",
+  "license": "Apache-2.0",
   "supports": "!uwp",
   "dependencies": [
     {

--- a/versions/a-/apr.json
+++ b/versions/a-/apr.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9fd8035a5cce96ad1b89000ac34431551ca39458",
+      "version": "1.7.0",
+      "port-version": 8
+    },
+    {
       "git-tree": "e73f548920cc5add8a47af9b8775a26c89050101",
       "version": "1.7.0",
       "port-version": 7

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -110,7 +110,7 @@
     },
     "apr": {
       "baseline": "1.7.0",
-      "port-version": 7
+      "port-version": 8
     },
     "apr-util": {
       "baseline": "1.6.1",

